evm-contracts/src/v0.6/EACAggregatorProxy.sol

```diff
@@ -254,7 +254,9 @@ contract EACAggregatorProxy is AggregatorProxy {
    * contract or is the contract itself.
    */
   modifier checkAccess() {
-    require(accessController.hasAccess(msg.sender, msg.data), "No access");
+    AccessControllerInterface ac = accessController;
+    bool disabled = address(ac) == address(0);
+    require(disabled || ac.hasAccess(msg.sender, msg.data), "No access");
```

Comment:
You can save 5 gas by removing the line above:
```suggestion
    require(address(ac) == address(0) || ac.hasAccess(msg.sender, msg.data), "No access");
```
:joy:

Reply:
Adds up.